spec/unit/webrtc/groupCall.spec.ts

```diff
@@ -882,11 +884,27 @@ describe("Group Call", function () {
                 expect(await groupCall.setMicrophoneMuted(false)).toBe(false);
             });
 
+            it("returns false when no permission for audio stream", async () => {
+                const groupCall = await createAndEnterGroupCall(mockClient, room);
+                jest.spyOn(mockClient.getMediaHandler(), "getUserMediaStream").mockRejectedValueOnce(
+                    new Error("No Permission"),
+                );
+                expect(await groupCall.setMicrophoneMuted(false)).toBe(false);
```

Comment:
Shouldn't it return `true` since the track stays muted?

Reply:
The return value of the method is not the current state of mute. The return value refers to the successful execution of the method. The mute state is propagated via an event.

Reply:
Argh, that is confusing - the method on `call.ts` with the same name returns the new mute state...